packages/custom-storage/src/sql/SQLStorage.ts

```diff
@@ -390,41 +394,41 @@ export class SQLStorage {
     overwriteDatasets: boolean,
     isPrivateVis: boolean) {
 
-    for (const dataset of datasets) {
-      let tableName: string;
+      for (const dataset of datasets) {
```

Comment:
Not sure why we have these code indentation changes. IMO we should automate this on committing, in order to avoid this kind of formatting issues